	* ppds.py (ppdMakeModelSplit): Don't strip " Printer" from the end
	of the ppd-make-and-model string.  Generic printers should be
	named like "Generic XYZ Printer".
	Fixes https://fedorahosted.org/system-config-printer/ticket/33.

Index: system-config-printer-1.0.0/ppds.py
===================================================================
--- system-config-printer-1.0.0.orig/ppds.py	2008-06-22 17:39:04.554818490 +0200
+++ system-config-printer-1.0.0/ppds.py	2008-06-22 17:40:25.118819921 +0200
@@ -139,7 +139,6 @@ def ppdMakeModelSplit (ppd_make_and_mode
     model = re.sub (r"(?i)\s*\bPXL", "", model)
     model = re.sub (r"(?i)[\s_-]+BT\b", "", model)
     model = re.sub (r"(?i)\s*\(Bluetooth\)", "", model)
-    model = re.sub (r"(?i)\s+Printer\b", "", model)
     model = re.sub (r"(?i)\s*-\s*(RC|Ver(|sion))\s*-*\s*[0-9\.]+", "", model)
     model = re.sub (r"(?i)\s*-\s*(RC|Ver(|sion))\b", "", model)
     model = re.sub (r"(?i)\s*-\s*$", "", model)
